projects/1.19/assets/serene-seasons/sereneseasons/lang/zh_cn.json

```
@@ -0,0 +1,38 @@
+{
+  "commands.sereneseasons.usage": "/season <get/set> [args]",
+  "commands.sereneseasons.getseason.success": "目前的季节为 %s, 天 %s/%s, 刻 %s/%s",
+  "commands.sereneseasons.setseason.success": "设定季节为 %s",
+  "commands.sereneseasons.setseason.fail": "无效的季节 %s",
+  "commands.sereneseasons.setseason.disabled": "季节更迭已被禁用!",
```

```suggestion
  "commands.sereneseasons.getseason.success": "当前季节为%s：第%s/%s天，第%s/%s刻",
  "commands.sereneseasons.setseason.success": "季节已设定为%s",
  "commands.sereneseasons.setseason.fail": "季节%s无效",
  "commands.sereneseasons.setseason.disabled": "季节更替已禁用！",
```
见库内1165及1182版本的Serene Seasons
同时请注意1165及以上版本无需再在全角字符和半角字符间添加空格（%s也不一定就是半角字符），所有标点符号请用全角

---

个人感觉
"commands.sereneseasons.setseason.success": "已将季节设为%s",
"commands.sereneseasons.setseason.disabled": "已禁用季节更替！",
读起来更舒服
当然如果要像我这么改那得把别的版本也改了